### ISSUE TITLE
[Dev] Traveler's Backpack compatibility help request

### ISSUE THREAD

**Issue by Tiviacz1337** [Globox1997/Dehydration#125]
### Make sure you are not opening a duplicate.

- [X] I have searched the issue tracker and did not find an issue describing my bug.

### Description of your suggestion.

Hello, I'm a developer of Traveler's Backpack. I'd like to add compatibility with Dehydration mod, but I'm slightly confused about hydration values that would be added on drinking fluids. 
If you could help me with that I'd be glad since I want to make compat that would work in a similiar way like in original mod. 
Basically I'd like to add hydration to following fluids: 
Regular water (1 Bucket on drink)
Purified water (1 Bucket on drink) 
Potions (1 Bottle on drink)
Milk (1 Bucket on drink)
I'd need hydration values of that fluids that are used when players drink those fluids from regular items
